openbb_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -44,6 +44,10 @@
 
 logger = logging.getLogger(__name__)
 
+no_ticker_message = (
```

Comment:
shouldn't the cmds under this menu also be grey if ticker is not loaded?

Reply:
So the reason they aren't is because they can all be run with cmd -t